docs/fields.md

```diff
@@ -432,7 +436,9 @@ class MyModel(edgy.Model):
 * **related_name** - The name to use for the relation from the related object back to this one.
 * **through** - The model to be used for the relationship. Edgy generates the model by default
                 if None is provided or **through** is an abstract model.
-* **embed_through** - When traversing, embed the through object in this attribute. Otherwise it is not accessable.
+* **embed_through** - When traversing, embed the through object in this attribute. Otherwise it is not accessable from the result.
+                      if empty, the old behaviour is used to query from the through model as base
+                      If not an empty string or False this is only possible from string as prefix or not accessable with False.
```

Comment:
Is tge through still available? Because we have also the note. This needs to be properly documented